challenge_3/python/psonghi/src/majority_element.py

```diff
@@ -0,0 +1,9 @@
+def majority_element(array):
+    half_time = len(array)/2
+    for element in array:
+        if array.count(element) > half_time:
```

Comment:
Remember that one of the goals of this challenge is to implement it in O(n) complexity. 

With the counter class mentioned previously you should have no problem reducing this from O(n^2) to O(n)

Good Luck